portal-api/src/v1/gef/routes.js

```diff
@@ -16,70 +17,70 @@ const router = express.Router();
 // Application
 router
   .route('/application')
-  .get(validate({ role: ['maker', 'checker', 'data-admin', 'admin'] }), application.getAll)
-  .post(validate({ role: ['maker', 'data-admin'] }), application.create);
+  .get(validateUserHasAtLeastOneAllowedRole({ allowedRoles: [MAKER, CHECKER, READ_ONLY, DATA_ADMIN, ADMIN] }), application.getAll)
```

Comment:
To confirm `DATA_AMIN` will be removed after?

Reply:
Yep that's right :)